src/BenchmarkDotNet/Disassemblers/DisassemblyDiagnoser.cs

```diff
@@ -166,7 +166,7 @@ private class NativeCodeSizeMetricDescriptor : IMetricDescriptor
             public string DisplayName => "Code Size";
             public string Legend => "Native code size of the disassembled method(s)";
             public string NumberFormat => "N0";
```

Comment:
`"0.##"` for consistency with memory?

Reply:
In the second today commit I changed it to correct double(second table), but in the third commit I've back it back(third table), because `CodeSize` is always integer.

Reply:
Allocations are always integer also, they are both measured in bytes. I think they should be consistent with their output. I like what you did with the fractional values in KB/MB/etc and I think the CodeSize should do the same.